test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+// <reference types="Cypress" />
+const helpers = {
+    load(additionalProps = { canUpload: true, permissions: { canUpload: true } }) {
+        cy.visit('/Elements/ContentExplorer', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+    },
+    getRow(rowNum) {
+        return cy.getByDataPreview('ContentExplorer').find(`.bce-item-row-${rowNum}`);
+    },
+    checkRows(rowNum) {
+        cy.getByDataPreview('ContentExplorer')
+            .find('.ReactVirtualized__Table')
+            .then(table => {
```

Comment:
Similarly, we could instrument the Table with a data-testid instead

Reply:
Same situation as above in that I don't think it's possible to add a data-testid to something inside the react-virtualized library.  I was able to add data-testid's to everything else in the file though.

Reply:
It seems like this could be simplified quite a bit by using more advanced chai or [chai-jquery selectors](https://docs.cypress.io/guides/references/assertions.html#Chai-jQuery), such as `.descendants`.